dotcom-rendering/src/client/ophan/ophan.ts

```diff
@@ -143,3 +143,14 @@ export const recordPerformance = async (): Promise<void> => {
 		performance,
 	});
 };
+
+const experiencesSet = new Set(['dotcom-rendering']);
+export const recordExperiences = async (
+	...experiences: string[]
+): Promise<void> => {
+	for (const experience of experiences) {
+		experiencesSet.add(experience);
+	}
+	const { record } = await getOphan();
+	record({ experiences: [...experiencesSet].join(',') });
```

Comment:
it's ok to send this multiple times, is that correct? and does each subsequent `record` override the previous one?

Reply:
Yes, it can be sent multiple times, but the last write wins.